docs/data/install-xrootd.md

```diff
@@ -290,6 +290,70 @@ fermicloud054.fnal.gov complete inventory as of Tue Apr 12 07:38:29 2011 /data/x
 
 **Note**: In this example, `fermicloud53.fnal.gov` is a redirector node and `fermicloud054.fnal.gov` is a data node.
 
+### (Optional) Enabling Xrootd over HTTP
+
+XRootD can be accessed using the HTTP protocol. To do that:
+
+1.   Modify `/etc/xrootd/xrootd-clustered.cfg` and add the following lines:
+
+        :::file
+           sec.protocol /usr/lib64 gsi \
```

Comment:
This line is correct for enabling authorization, but is unrelated to HTTP / HTTPS.  We should find the xrootd-lcmaps documentation and reference that -- make sure that doc uses the same line too!

Reply:
sure